lib/net/authorize/api/contract/v1/ARBSubscriptionMaskedType.php

```diff
@@ -47,6 +47,12 @@ class ARBSubscriptionMaskedType
      */
     private $order = null;
 
+    /**
+     * @property \net\authorize\api\contract\v1\TransactionDetailsType[]
+     */
+    private $arbTransactions = null;
```

Comment:
This should be of a new type \net\authorize\api\contract\v1\ARBTransactionList . New class for this type required.